nil/services/rpc/rawapi/local_txnpool.go

```diff
@@ -27,11 +27,11 @@ func (api *LocalShardApi) SendTransaction(ctx context.Context, encoded []byte) (
 }
 
 func (api *LocalShardApi) GetTxpoolStatus(ctx context.Context) (uint64, error) {
-	return uint64(api.txnpool.GetQueue().Len()), nil
+	return uint64(api.txnpool.GetPendingLength()), nil
 }
 
 func (api *LocalShardApi) GetTxpoolContent(ctx context.Context) ([]*types.Transaction, error) {
-	txns, err := api.txnpool.Peek(api.txnpool.GetQueue().Len())
+	txns, err := api.txnpool.Peek(1_000_000)
```

Comment:
txpool has "Size" field in config. And you can try to use this field instead of hardcoded constant.